unity-client/Assets/Scripts/MainScripts/DCL/Models/Quests/QuestReward.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+[System.Serializable]
+public class QuestReward
+{
+    public string id;
+    public string name;
+    public string type;
+    public string imageUrl;
+    public string status;
```

Comment:
Can this be an enum? If it an, we should change QuestModel static reward status string as well

Reply:
As far as I know, it cannot be an enum if we wanna keep using the way faster Unity's `ToJson/FromJson` utility. It doesn't parse enums